db/migrate/20150107233846_create_oauth_access_grants.rb

```diff
@@ -0,0 +1,15 @@
+class CreateOauthAccessGrants < ActiveRecord::Migration
```

Comment:
But it uses strings rather than symbols :sad:

Does it check for the existing migration?